src/chi-vue/src/components/data-table/data-table.scss

```diff
@@ -26,6 +26,26 @@
   }
 }
 
+.chi-data-table__toolbar {
```

Comment:
Temporary CSS, will be removed once the HTML markup is ready. Related to FSDE-306